db/migrate/20200406113832_drop_variable_lookup_key.rb

```diff
@@ -0,0 +1,5 @@
+class DropVariableLookupKey < ActiveRecord::Migration[5.2]
+  def change
+    LookupKey.where(:type => 'VariableLookupKey').destroy_all
```

Comment:
I think this will fail if there are such lookup keys since rails will attempt to load them before destroying but will fail due to missing model. you can use delete_all instead, but you'll need to take care of the associated records manually:
```suggestion
    variables = LookupKey.where(:type => 'VariableLookupKey')
    LookupValue.where(lookup_key: variables).delete_all
    variables.delete_all
```